### PR TITLE
Adding examples of the new Cloud Bigtable+Dataflow integration.

### PR DESCRIPTION
**Pull request by sduskis**
The README.md needs a bit of work.
We should also setup the pom.xml to properly configure exec:exec for the three examples... perhaps a profile per example?

Feel free to hold off on the review and merge, or to do it and change things appropriately

### CODE REVIEW

File: java/dataflow-connector-examples/pom.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (lesv) [inline java/dataflow-connector-examples/pom.xml]: Needs an apache license.

File: java/dataflow-connector-examples/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Also, you need the following command line options for Bigtable:
+
+1. --bigtableProject=some_project 
+2. --bigtableClusterId=cluster_name 
+3. --bigtableZone=us-central1-b
+4. --bigtableTable=someTableName
+
+You can run this via maven, for example:
+
+`mvn exec:java -Dexec.mainClass="com.google.cloud.dataflow.sdk.io.bigtable.example.HelloWorldParDo" -Dexec.args="--runner=BlockingDataflowPipelineRunner --project=some_project --stagingLocation=gs://some_bucket --bigtableProject=some_project --bigtableClusterId=cluster_name--bigtableZone=us-central1-b --bigtableTable=someTableName"`
```
Comment (lesv) [inline java/dataflow-connector-examples/README.md]: a few things I'll fix -- parameter names mostly have Id at the end like bigtableZoneId, and there is a missing space between clusterID & zone

**Comment by lesv** [comment]
The office is insanely noisy, so I can't fix this evening, if you could add the license, so I can LGTM, I'll fix the readme in the AM.

File: java/dataflow-connector-examples/pom.xml (hunk 1 above)
Comment (sduskis) [inline java/dataflow-connector-examples/pom.xml]: done.

**Comment by lesv** [comment]
LGTM - the README still needs work, I'll take that on.

**Pull request merged**